src/docserv/bih.py

```diff
@@ -0,0 +1,389 @@
+import json
```

Comment:
Actually, it's for the `__init__.py` file (GitHub doesn't allow me to add a comment on an empty file, grrr):

I would add the following information:

* `__version__` contains the version of docserv²
* `__author__` your name. :wink: